src/web/app/pages-logs/logs-page.component.html

```diff
@@ -0,0 +1,70 @@
+<h1>Logs</h1>
+<p>This page allows you to view the logs in the past {{ LOGS_RETENTION_PERIOD_IN_DAYS }} days.</p>
+<hr/>
+
+<div *tmIsLoading="isLoading" class="card bg-form no-border">
+  <div class="card-body">
+    <div class="row form-group">
+      <div class="col-6">
+        <div class="col-12 mb-1">
+          <label for="logs-from-datepicker" class="font-weight-bold">Search period from</label>
+        </div>
+        <div class="input-group">
+          <div class="col-md-6 input-group">
+            <input id="logs-from-datepicker" type="text" class="form-control" ngbDatepicker [minDate]="earliestSearchDate" [maxDate]="dateToday" [(ngModel)]="formModel.logsDateFrom" #logsFromDp="ngbDatepicker"/>
+            <button class="btn btn-light" (click)="logsFromDp.toggle()" type="button">
+              <i class="fas fa-calendar-alt"></i>
+            </button>
+          </div>
+          <div id="logs-from-timepicker" class="col-md-3">
+            <tm-time-picker [(time)]="formModel.logsTimeFrom"></tm-time-picker>
+          </div>
+        </div>
+      </div>
+      <div class="col-4">
+        <div class="col-12">
+          <label for="log-severity-checkbox" class="font-weight-bold">Log severity level</label>
+        </div>
+        <div class="form-check" *ngFor="let severity of SEVERITIES">
+          <label class="form-check-label" id="log-severity-checkbox">
+            <input type="checkbox" class="form-check-input"
+                   [checked]="formModel.logsSeverity.has(severity)" (click)="toggleSelection(severity)">
+            <span>{{ severity }}</span>
+          </label>
+        </div>
+      </div>
+      <div class="col-1 mt-4">
+        <button id="query-button" class="float-right btn btn-primary" (click)="searchForLogs()">Query</button>
+      </div>
+    </div>
+    <div class="row form-group">
+      <div class="col-6">
+        <div class="col-12 mb-1">
+          <label for="logs-to-datepicker" class="font-weight-bold">Search period until</label>
+        </div>
+        <div class="input-group">
+          <div class="col-md-6 input-group">
+            <input id="logs-to-datepicker" type="text" class="form-control" ngbDatepicker [minDate]="earliestSearchDate" [maxDate]="dateToday" [(ngModel)]="formModel.logsDateTo" #logsToDp="ngbDatepicker"/>
+            <button class="btn btn-light" (click)="logsToDp.toggle()" type="button">
+              <i class="fas fa-calendar-alt"></i>
+            </button>
+          </div>
+          <div id="logs-to-timepicker" class="col-md-3">
+            <tm-time-picker [(time)]="formModel.logsTimeTo"></tm-time-picker>
+          </div>
+        </div>
+      </div>
+    </div>
+  </div>
+</div>
+
+<div *tmIsLoading="isSearching">
+  <div *ngIf="hasResult">
```

Comment:
It will just be empty since the user has not done any query yet (no table, no message). If there are no logs in the time period queried, then a message "There are no logs in this time period." will be displayed.